evgMrmApp/Db/evgEvtClk.db

```diff
@@ -157,3 +157,51 @@ record(bi, "$(P)Pll-Sts") {
   field(ONAM, "OK")
   field(ZSV , "MAJOR")
 }
+
+record(mbbo, "$(P)PLL-Bandwidth-Sel") {
+    field( DESC, "EVG Evt Clock Bandwidth")
+    field( DTYP, "Obj Prop uint16")
+    field( OUT,  "@OBJ=$(OBJ), PROP=PLL Bandwidth")
+    field( PINI, "YES")
+    field( VAL,  "$(PLL-Bandwidth-Sel\=2)")
```

Comment:
Is that a typo ? this `\` before the `=`

Reply:
Also, it seems to be standard to have the macro names on UPPERCASE, it might be good to keep this standard.

Reply:
I used the same code in Tony's original PR. I think the "\=" he used may be a standard for older versions of EPICS, or some other reasons. I saw him using it in other places. 
I also updated the macro name to PLLBW.